### ISSUE TITLE
GSoC 2019: Use this issue for discussion

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#12]
Please use this issue for any Google Summer of Code-related questions and discussion that does not fit elsewhere.

**Comment by freddieee** [comment]
Hi John, I would like to contribute to this project. According to your advice I am trying to improve the test coverage as the first step. But I found it was a  little tricky to get the test coverage report for all modules since test files and module files are in different directories. Do you think it is a good idea a write a script file in the directory "tests" to generate coverage report automatically?

**Comment by freddieee** [comment]
Sorry, I i miss the ```.coveragerc``` file in the root, now I can see the report

**Comment by pdxjohnny** [comment]
No worries! I've created #27 and will add more instructions on how to run coverage, because I forgot until you said this that it took me a bit to get the hang of it, so I'll add docs explaining.

**Comment by yashlamba** [comment]
Hey!
Can I help with some documentation part of the project? It will help me understand the project even better.

**Comment by pdxjohnny** [comment]
Yes @yashlamba ! I've created #35 so that we have an issue to discuss docs stuff.
